packages/themes/pennwell/templates/search.marko

```diff
@@ -0,0 +1,24 @@
+import getAdUnit from '@endeavorb2b/base-website-common/utils/gam/adunit-website-section';
+
+$ const { site, config } = out.global;
+$ const { engine, apiKey, domain } = site.getAsObject('search');
+
+<theme-pennwell-search-layout>
+  <cms-page-container for="search" class="container-fluid-max">
+    <endeavor-gam-ad-unit-display id="gpt-ad-lb1" modifiers=["top-of-page"] />
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col">
+          <h1>Search ${config.get('siteName')}</h1>
+          <hr>
+          <if(engine === 'google')>
+            <endeavor-search-google apiKey=apiKey />
+          </if>
+          <else-if(engine === 'bing')>
+            <endeavor-search-bing apiKey=apiKey domain=domain />
```

Comment:
Change to `<cms-browser-component name="SearchIntegrationBing" props={ apiKey, domain } />`